src/depends-on.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"os"
+	"time"
+)
+
+func main() {
+
+	services := os.Args[1:]
```

Comment:
I would also print a simple help message if the user runs this command with no arguments or passes `-h|--help`

Reply:
@meshok0 I'm sure there is an arguments parser library for go, so there might be a good idea to use it instead of parsing the args directly.